shadowsocks-libev-debian.sh

```diff
@@ -279,7 +279,7 @@ pre_install(){
             break
         fi
     fi
-    echo -e "[${red}Error${plain}] Please enter a correct number [1-65535]"
+    echo -e "[${red}Error${plain}] Please enter a port number (Recommended 80 or 443) [1-65535]"
```

Comment:
> (Recommended 80 or 443)
Definitely not recommended. With 443 port, IP will be banned in 30 minutes or less.

Reply:
which ones do u recommend?

Reply:
above 10000

Reply:
> which ones do u recommend?

above 10000

Reply:
1-65535
Choose a coupe that u like and u would recommend, cause I feel like people just leave it as default and it's not hard to find..